scripts/add_a_control_method.sh

```diff
@@ -0,0 +1,8 @@
+component_name="my_control_method"
```

Comment:
Add shebang 

```suggestion
#!/bin/bash
component_name="my_control_method"
```

Do this for other scripts, and make sure the other executables are executable (`chmod +x scripts/*`)